examples/android/xmtpv3_example/.gitignore

```diff
@@ -36,4 +36,5 @@ google-services.json
 jniLibs/
 xmtp_dh.kt
 xmtpv3.kt
-MainActivity.kt
\ No newline at end of file
+MainActivity.kt
+ExampleInstrumentedTest.kt
```

Comment:
Curious why we need to .gitignore this file. Is that because it gets copied over as part of the build?